tests/check_consistent.py

```diff
@@ -83,6 +83,21 @@ def check_stubs() -> None:
             assert not py_files_present, error_message
 
 
+def check_distutils() -> None:
+    """Check whether all setuptools._distutils files are re-exported from distutils."""
+    old_cwd = Path.cwd()
+    try:
+        os.chdir(Path("stubs", "setuptools", "setuptools", "_distutils"))
+        all_setuptools_files = set(Path(".").rglob("*.pyi"))
+        os.chdir(old_cwd / "stubs" / "setuptools" / "distutils")
+        all_distutils_files = set(Path(".").rglob("*.pyi"))
+    finally:
+        os.chdir(old_cwd)
```

Comment:
What about doing something like this? Slightly simpler, and means that we don't need to do the `os.chdir` stuff:

```suggestion
    def all_relative_paths_in_directory(path: str) -> set[Path]:
        return {pyi.relative_to(path) for pyi in Path(path).rglob("*.pyi")}

    all_setuptools_files = all_relative_paths_in_directory("stubs/setuptools/setuptools/_distutils")
    all_distutils_files = all_relative_paths_in_directory("stubs/setuptools/distutils")
```